doc/connectivity/bluetooth/api/mesh/brg_cfg.rst

```diff
@@ -113,13 +113,11 @@ protection to ensure network security. Key considerations to take into account a
 Relay buffer considerations
 ===========================
 
-When a message is relayed between subnets by a Subnet Bridge, it is allocated from the relay buffer.
-To ensure that messages can be retransmitted to all subnetworks,
-the :kconfig:option:`CONFIG_BT_MESH_RELAY_BUF_COUNT` option should be increased accordingly.
-
-However, if the :kconfig:option:`CONFIG_BT_MESH_RELAY` feature is disabled, the messages will be
-allocated from the advertising buffer instead. In this case, increase the
-:kconfig:option:`CONFIG_BT_MESH_ADV_BUF_COUNT` option to allow for sufficient buffer space.
+When a message is relayed between subnets by a Subnet Bridge, it is allocated from the relay buffer
+pool and transmitted using the relay advertising sets. These will be used even if the relay feature
```

Comment:
Reading this isolated, I am missing half a sentence what these buffer and advertising sets are used for when MESH_RELAY is disabled... not for relaying, right?

Reply:
The relay and bridge functionality is quite similar IMO, with relaying you're a middleman for nodes in the same subnet, but with subnet bridge you're a middleman between subnets. So it's _kind of_ relaying. The `bt_mesh_net_relay` func is used for both: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/net.c#L685